packages/react-native/scripts/cocoapods/utils.rb

```diff
@@ -73,6 +73,20 @@ def self.set_use_hermes_build_setting(installer, hermes_enabled)
         end
     end
 
+    def self.set_swift_compilation_conditions(installer)
```

Comment:
Maybe we could create a utility to set any flag by name and value because for any flags in the future this code will be redundant